CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Expose bias flag for feedforwards, same default as Timm [#220]
 - Update eps value for layernormm, same default as torch [#221]
 - PreNorm bugfix, only one input was normalized [#233]
+- Fix bug where embedding dimensions that did not match model dim would lead to a crash [#244]
```

Comment:
this would need to be moved up, it would be part of the next release ? I'm kind of on the fence as of whether this is a "bug" (there are plenty of settings which you can pass which will "crash", in that they will tell you down the line that the dimensions mismatch), but as you see fit

Reply:
Ah yeah, not a bug, sorry! Feel free to rename it as you see fit.